src/lib.rs

```diff
@@ -24,7 +33,14 @@ mod session_store;
 mod session_timers;
 
 pub use config::{AxumSessionConfig, AxumSessionMode, Key, SameSite};
+#[cfg(feature = "DatabaseTrait")]
```

Comment:
the Trait should not be behind a feature itself since it is being used by the others supported database type.